src/requests.jl

```diff
@@ -285,6 +285,6 @@ See also [`request_json`](@ref) and [`request_raw`](@ref).
 
     @logmsg LogLevel(-1_000) "FHIRClient.request()" path verb tryparse_json(response_body)
 
-    response_object = JSON3.read(response_body, T; kwargs...)::T
+    response_object = JSON3.read(response_body, T; kwargs...)
```

Comment:
It doesn't hurt to also have this type assertion here, does it?

```suggestion
    response_object = JSON3.read(response_body, T; kwargs...)::T
```